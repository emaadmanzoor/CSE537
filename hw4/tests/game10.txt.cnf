c tests/game10.txt
p cnf 3 6
-3 0
-2 0
-1 0
1 0
2 0
3 0
