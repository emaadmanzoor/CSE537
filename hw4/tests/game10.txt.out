c tests/game10.txt
p cnf 3 0
